buildpipeline/pipeline.json

```diff
@@ -158,7 +158,8 @@
         {
           "Name": "Core-Setup-Signing-Windows-x64",
           "Parameters": {
-            "PB_PortableBuild": "-portable"
+            "PB_PortableBuild": "-portable",
+            "SIGNED_PACKAGES": "false"
```

Comment:
I recommend setting it to false by default, and only setting it to true for the nonportable x64 build. The x86 entries below should also both be false.